qdrant-landing/content/documentation/guides/security.md

```diff
@@ -132,6 +130,209 @@ export QDRANT__SERVICE__READ_ONLY_API_KEY=your_secret_read_only_api_key_here
 
 Both API keys can be used simultaneously.
 
+### Granular access control with JWT
+
+*Available as of v1.9.0*
+
+For more complex cases, Qdrant supports granular access control with [JSON Web Tokens (JWT)](https://jwt.io/). This allows you to have [Role-based access control (RBAC)](https://en.wikipedia.org/wiki/Role-based_access_control) integrated into the system. In this way, you can define roles and permissions for users and restrict access to sensitive endpoints.
+
+To enable JWT-based authentication in your own Qdrant instance you need to specify the `api-key` and enable the `jwt_rbac` feature in the configuration:
+
+```yaml
+service:
+  api_key: you_secret_api_key_here
+  jwt_rbac: true
+```
+
+Or with the environment variables:
+
+```bash
+export QDRANT__SERVICE__API_KEY=your_secret_api_key_here 
+export QDRANT__SERVICE__JWT_RBAC=true
+```
+
+The `api_key` you set in the configuration will be used to encode and decode the JWTs, so –needless to say– keep it secure. If your `api_key` changes, all existing tokens will be invalid.
+
+To use JWT-based authentication, you need to provide it as a bearer token in the `Authorization` header, or as an key in the `Api-Key` header of your requests.
+```json
+{ "Authorization": "Bearer <JWT>" }
+// or 
+{ "Api-Key": "<JWT>" }
+```
+
+#### Generating JSON Web Tokens
+
+Anyone who knows the `api_key` can generate tokens.
```

Comment:
Also, I think it is worth mentioning explicitly that:

- An user don't need to access to the Qdrant instance to generate the key.
  It could be generated offline using any of the existing libraries and tools.

- Qdrant Web UI provides a convenient UI to generate keys.